### PR TITLE
Updated docs to reflect the new "renderExperience" call as the main rendering entrypoint 

### PR DESCRIPTION
**Pull request by webbergreg**

### CODE REVIEW

**Review by jbmcg — CHANGES_REQUESTED** [review]
Might make sense to replace the "These values will be provided to you by the Imposium team when your account and new projects are created." line near the top of the Client section with some info about where to find the access key / story ID / composition ID in the editor

File: docs/www/client.md [hunk 1]
```diff
@@ -328,6 +316,48 @@ var inventory = {
 client.createExperience(inventory);
 ```
 
+#### getExperience
+
+`Imposium.Client.getExperience(experienceId: String): void`
+
+Gets experiences by id and triggers the [`GOT_EXPERIENCE`](#GOT_EXPERIENCE) event on success. If processing an experience is intentionally deferred until after an experience is created (i.e: rendering is invoked by the first visit to a deeplink) making this call will invoke processing.
```
[the remark this answers is not in the document]
Comment (webbergreg) [reply docs/www/client.md]: Change the initial copy a bit and fixed the i.e. issue

File: docs/www/client.md [hunk 2]
```diff
@@ -231,56 +197,65 @@ function gotExperience(experience) {
...
-// Existing experience ID
-var myExperienceId = 'bdc8ca2e-6c68-4c37-8396-bae2c65964d9';
 var videoElement = document.getElementById('my-video-element');
+var textInput = document.getElementById('my-text-input');
+var fileInput = document.getElementById('ny-file-input');
 
 // Executes when the experience is retrieved or done processing
 function gotExperience(experience) {
-     videoElement.src = data.output.videos['mp4_720'].url;
+    videoElement.src = data.output.videos['mp4_720'].url;
```
[the remark this answers is not in the document]
Comment (webbergreg) [reply docs/www/client.md]: Changed it to [0] anywhere it's using "renderExperience"

File: docs/www/client.md [hunk 3]
```diff
@@ -231,56 +197,65 @@ function gotExperience(experience) {
...
 
-Gets experiences by id and triggers the [`GOT_EXPERIENCE`](#GOT_EXPERIENCE) event on success. If processing an expereince is intentionally deferred until after an experience is created (i.e: rendering is invoked by the first visit to a deeplink) making this call will invoke processing.
-
-Note: If you are using [`Imposium.Player`](/player) to handle playback then you can skip setting [`GOT_EXPERIENCE`](#GOT_EXPERIENCE) since the player will always consume processed experiences for you. It should be said though that setting this event can still be useful when using the player if you wish to update application state or markup when an experience becomes ready (i.e: hiding a 'loading' or 'now processing' view).
+[`renderExperience`](#renderExperience) Is the easiest way to render a dynamic video and immediatly use the rendered video files. It creates a new instance of an experience, automatically kicks off the render, and will fire [`GOT_EXPERIENCE`](#GOT_EXPERIENCE) once the render is complete. [`renderExperience`](#renderExperience) is essentially a combination of [`createExperience`](#createExperience) and [`getExperience`](#getExperience), in a single easy to use call.
```
[the remark this answers is not in the document]
Comment (webbergreg) [reply docs/www/client.md]: fixed

File: docs/www/client.md [hunk 4]
```diff
@@ -214,6 +146,40 @@ function errorHandler(error) {
 
 client.on(Imposium.Events.ERROR, errorHandler);
 ```
+
+##### EXPERIENCE_CREATED
+
+**Callback Signature:** `function experienceCreated(experience: Object)`
+
+Triggered when an experience record is first created. This is helpful in SPA environments for use cases where writing the id for an experience to a url can be used for SEO purposes, sharing, etc. This event can also be useful for updating loading markup (i.e: uploading has finished, moving onto processing...) as it means any dynamic data that was submitted to our web servers by a consumer was effectively transmitted.
```
[the remark this answers is not in the document]
Comment (webbergreg) [reply docs/www/client.md]: fixed.

File: docs/www/basic_tutorial.md [hunk 5]
```diff
@@ -96,8 +96,8 @@ form.addEventListener('submit', function(e) {
...
 
     // Consumes dynamicData and begins process of listening for video data 
-    client.createExperience(dynamicData);
+    client.renderExperience(dynamicData);
 });
 ```
 
-The above snippet will handle the end to end flow of creating video with Imposium, including playback. When `Imposium.client.createExperience` is called the client beings listening for processed video data. Once the data becomes available the video player will begin streaming it.
\ No newline at end of file
+The above snippet will handle the end to end flow of creating video with Imposium, including playback. When `Imposium.client.renderExperience` is called the client beings listening for processed video data. Once the data becomes available the video player will begin streaming it.
```
[the remark this answers is not in the document]
Comment (webbergreg) [reply docs/www/basic_tutorial.md]: fixed.

**Review by jbmcg — APPROVED** [review]

**Pull request merged**